exercises/concept/train-driver/.docs/instructions.md

```diff
@@ -0,0 +1,111 @@
+# Instructions
+
+Your friend is a train driver and has to drive cargo trains between cities.
+Although your friend isn't amazing with handling the computers and would like some help with it.
+Your friend would like your help organizing the train and correcting the mistakes in the data.
+
+
+```exercism/note
+To practice, use a for the rest or spread to solve each of the tasks below.
+```
+
+## 1. Convert the data to an array
+
+Your friend has been keeping track of how much each wagon weighs. Although they are not sure how many wagons and would like the data to be returned as an array.
+
+```exercism/note
```

Comment:
Note blocks should not be used for the normal instructions of the exercise, only for special things that need to stand out (like the note at the top about using rest and spread).

Applies to all the tasks.